### ISSUE TITLE
Truffle 4.0 Release Checklist

### ISSUE THREAD

**Issue by tcoulter** [trufflesuite/truffle#646]
- [x] Official Release!
  - [x] Flush PRs waiting on CI (incl. `truffle-config` release)
  - [x] Merge `develop`s -> `master`s
  - [x] Pre-flight checks (workspace on `master`, symlinked, `cli.bundled.js`)
  - [x] Version bumps, dependency upgrades, npm publishings
  - [x] Post-flight checks
- [x] Release Notes
- [x] New `truffle develop` gif (NICE TO HAVE)
- [x] Mailer
- [x] Twitter
- [x] Reddit

**Comment by gnidan** [comment]
Done!
